rust/yarp-sys/tests/unescape_tests.rs

```diff
@@ -0,0 +1,79 @@
+use std::{ffi::CString, mem::MaybeUninit};
+
+use yarp_sys::{
+    yp_list_empty_p, yp_list_init, yp_list_t, yp_parser_init, yp_parser_t, yp_string_t,
+    yp_string_t_type, yp_unescape_calculate_difference, yp_unescape_manipulate_string,
+    yp_unescape_type_t,
+};
+
+#[test]
+fn unescape_manipulate_string_test() {
```

Comment:
`yp_unescape_` is internal stuff, I don't think it makes sense to expose that to Rust.
(it's exposed under YARP::Debug for YARP internal testing but that's it, it's not meant to be used by anything but YARP itself)

Reply:
My intent was to follow the `YP_EXPORTED_FUNCTION`s, but it sounds like that was a mistake. Is there a better way to deduce what's in the public API?